.changeset/fair-flowers-sleep.md

```diff
@@ -0,0 +1,7 @@
+---
+'astro': minor
```

Comment:
This definitely seems non-breaking to me. I think we should avoid forcing people to update their package.json version numbers as much as possible, and this seems like a clear case of added capability.

Reply:
Sweet! I'm good with that.